lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -135,9 +135,68 @@ function dosomething_mbp_update_7007() {
   dosomething_mbp_install_productions();
 }
 
+/**
+ * Add config for Mailsystem and Mandrill modules.
+ */
+function dosomething_mbp_update_7008() {
+  module_enable('mailsystem', 'mandrill');
+  $mailsystem_keys = array(
+    'default-system' => 'MandrillMailSystem',
+    'mandrill_user_register' => 'MandrillMailSystem',
+    'mandrill_user_password' => 'MandrillMailSystem',
+    'mandrill_campaign_signup' => 'MandrillMailSystem',
+    'mandrill_campaign_group_signup' => 'MandrillMailSystem',
+    'mandrill_campaign_reportback' => 'MandrillMailSystem',
+  );
+  variable_set('mail_system', $mailsystem_keys);
+  variable_set('mandrill_api_key', 'Zqk_59h4hMerx3ZlEvC13A');
```

Comment:
I'm not sure if we want to have the API key publicly available and set. What are your thoughts?

Reply:
We could always just manually set it after deployment. Unless you all have another method for handling API keys (env vars perhaps)?

Reply:
We're deploying this to Pantheon, right? Do they have anything like that available? Otherwise, yeah, I guess we could set this manually. If it's set manually after this is deployed, would anything in this update fail?

Reply:
Pantheon supports https://github.com/pantheon-systems/terminus-secrets-plugin which writes to a `secrets.json` file that you can read from, as well as https://pantheon.io/docs/guides/lockr/ which requires 2 new modules so it's probably overkill. 

I think just manual setting is probably fine though. Nothing will fail with the update - emails just won't send until we add the API key. That sound OK?

Reply:
Sounds good to me! I'll generate a new API key for Mandrill for the Indonesia site now.